tools/dbTools.py

```diff
@@ -56,54 +48,25 @@ def _execute(self, query, params=()):
             )
             raise
 
-    def _check_db(self):
-        """Create the tables needed by the web scraper if they don't exists."""
-        self._execute(
-            """
-            CREATE TABLE IF NOT EXISTS article
-            (
-                id SERIAL PRIMARY KEY,
-                url VARCHAR(255),
-                file_hash VARCHAR(32),
-                scrap_again BOOLEAN DEFAULT FALSE,
-                timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP
-            );
-            """
-        )
-        self._execute(
-            """
-            CREATE TABLE IF NOT EXISTS spiders
-            (
-                id SERIAL PRIMARY KEY,
-                name VARCHAR(64),
-                uuid VARCHAR(64),
-                status VARCHAR(255),
-                end_time TIMESTAMP,
-                start_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP
-            );
-            """
-        )
-        self.connection.commit()
```

Comment:
where does the table creation logic lives now? should it go to the db folder?

Reply:
I don't know actually. I'm sure this is not the right place, but I'm still thinking about it. I think we should go for some kind of `manage.py` file for all these kind of operations

Reply:
👍